.github/workflows/docs.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       matrix:
         branch:
           - master
+          - release-1.16
```

Comment:
We can't merge this yet because it would publish the docs as if the release has been completed.